NosTests/SocialGraphTests.swift

```diff
@@ -38,106 +38,6 @@ final class SocialGraphTests: CoreDataTestCase {
         try await eventually { await sut.followedKeys == expectedKeys }
     }
     
-    func testFollow() async throws {
```

Comment:
Weird. These shouldn't be deleted.

Reply:
That was me not understanding their purpose. They call some func that's _only_ called in tests and not in our production code. Additionally, they were expecting failure and _still_ failing in CI, so they've been disabled entirely for a little while. I don't know if we have a way to make them pass consistently in CI but if so, let's do it and reenable them, and if not, let's eliminate them.

Reply:
Restored for now... But I'd still like to fix or delete them.

Reply:
Oh, I am really confused on a few points.
> They call some func that's only called in tests and not in our production code.

Are you referring to the `eventually` helper function? I looked at the properties these are testing (`followedKeys` and `isInNetwork(:)`) and I see them being used in production code.

> Additionally, they were expecting failure and still failing in CI, so they've been disabled entirely for a little while.

I think we added the code to expect failures _in order to disable them_, and we're still planning to come back around and fix them. I like using `XCTExpectFailure` better than commenting out the test, renaming the function, or deleting it, because this way we make sure it continues to compile.

> I don't know if we have a way to make them pass consistently in CI

Are they still failing on CI even with `XCTExpectFailure`? Maybe we need more of them haha.

Reply:
@mplorentz yeah, even the ones with XCTExpectFailure were failing in CI. Not in the green-checkmark-expecting-failure kind of way, but in the red-x-even-though-we're-expecting-failure kind of way. So I disabled this and another test to make sure our PRs weren't blocked by failing tests:

<img width="334" alt="Screenshot 2024-03-20 at 9 44 45 AM" src="https://github.com/planetary-social/nos/assets/59564/8bd09cc4-5ade-4ada-9b87-c6b2aa086d5a">

(I think there's a better way to see what's disabled with Test Plans but we haven't migrated to those yet.)

Reply:
Ah, perfect! It showed itself right after we were talking about it: https://github.com/planetary-social/nos/actions/runs/8360178747/job/22885251067#step:5:112